packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -175,7 +218,8 @@ export function useDefaultViews( { postType } ) {
 						},
 					],
 				},
+				count: counts?.trash,
 			},
 		];
```

Comment:
I thought a follow up could filter out items that have no counts, e.g., `.filter( ( { count } ) => count > 0 )`


https://github.com/user-attachments/assets/796a0971-0949-43c0-a247-40a9bc0a68cb

Reply:
Hiding empty bundled views seems reasonable. Custom views should probably remain visible regardless.

Reply:
Custom views are unaffected. There is a snag however: it takes a tick longer to fetch the counts so you'll see a little delay on first load.

https://github.com/user-attachments/assets/6555f459-fcf7-4df1-aa00-ab4a06a7dcf6

A few options I can think of, some better than others:

1. Don't filter out `0` values. That means the list will appear in full, but the counts, including `0` items, will appear a little later. Maybe the `0` item could be unclickable/disabled/greyed out.
2. Show the full list, then filter out `0` values. That means the full list will appear, but `0` value items will _disappear_. 👎🏻 
3. Include a loading placeholder.
4. ?

Reply:
I've done 1 for now:

https://github.com/user-attachments/assets/4058ecb2-4d9c-47e0-9a48-de4591aab457

Reply:
A loading indicator would be a nice enhancement but can potentially come later.